Add -e option.
--- a/sl.c
+++ b/sl.c
@@ -53,6 +53,7 @@
 int LOGO      = 0;
 int FLY       = 0;
 int C51       = 0;
+int INTR      = 0;
 
 int my_mvaddstr(int y, int x, char *str)
 {
@@ -73,6 +74,7 @@
             case 'F': FLY      = 1; break;
             case 'l': LOGO     = 1; break;
             case 'c': C51      = 1; break;
+            case 'e': INTR     = 1; break;
             default:                break;
         }
     }
@@ -88,7 +90,9 @@
         }
     }
     initscr();
-    signal(SIGINT, SIG_IGN);
+    if (INTR == 0) {
+        signal(SIGINT, SIG_IGN);
+    }
     noecho();
     curs_set(0);
     nodelay(stdscr, TRUE);
